hack/build.sh

```diff
@@ -10,7 +10,7 @@ registry="${registry:-quay.io/confidential-containers}"
 name="cloud-api-adaptor-${CLOUD_PROVIDER}"
 tag=$(date +%Y%m%d%H%M%s)
 
-supported_arches=${ARCHES:-"linux/amd64,linux/s390x"}
+supported_arches=${ARCHES:-"linux/amd64"}
```

Comment:
Sorry to ask a basic question, but I assume the ibm cloud version will set `ARCHES=linux/amd64,linux/s390x`. I think that's the case in the GHA workflow, but do we need to support this in any scripts for 'manually' (ie not GHA) creating the CAA ibm cloud payload, or would we just doc this in the ibm cloud section?

Reply:
I think a doc in ibmcloud section is most appropriate